hugo/content/3_how-tos/31_account-management/309_user-management.md

```diff
@@ -41,16 +41,14 @@ To invite or add users to the rapyuta.io platform, do the following:
 
 3. Click **ADD USER**.
 
-The newly invited user is added to the existing list of **USERS**.
-
-Initially, the state of the user is **Invited**. 
-The invited user will receive an invitation email to join rapyuta.io.
-
-Once the invited user has registered and signed into rapyuta.io, their state
-changes to **Activated**.
+The newly invited user is added to the existing list of **USERS**. The **Account State** field displays the state of a user. A user can be in one of the following states:
+* **Invited**: The initial state of the user. The invited user will receive an invitation email to join rapyuta.io.
+* **Activated**: Once the invited user has registered and signed into rapyuta.io, their state automatically changes to **Activated**. Only an org admin can activate a deactivated user by selecting the activate icon under **Actions**. 
+* **Deactivated**: A user/organization admin can deactivate a user by selecting the deactivate icon under **Actions**.
+* **Suspended**: A user can be in the suspended state due to non-payment. |
 
 {{%notice info%}}
- The admin of an organization can view **Invited** users. All other users can only view **Activated** users.
+ The admin of an organization can view **Invited** and **Deactivated** users. All other users can only view **Activated** users.
```

Comment:
The admin of an organization can view user with any state. Only the other invited users are not visible to a non-admin user, all other states are visible